detection-rules/mass_campaign_recipient_address_new_sender.yml

```diff
@@ -0,0 +1,49 @@
+name: "Mass campaign: recipient address in subject, body, and link (first-time sender)"
+description: |
+  This detects a pattern commonly observed in mass spam and phishing campaigns.
+
+  The local_part or the full email address of the recipient is used in the subject,
+  body, and link query parameter to "personalize" the attack.
+references:
+  - "https://playground.sublimesecurity.com?id=d9143109-8886-4639-b634-d0a671848eb6"
+type: "rule"
+severity: "medium"
```

Comment:
this could also be "low", wdyt?

Reply:
"medium" feels right to me but could go either way